sample/dockerwait.js

```diff
@@ -13,11 +13,10 @@ var bootCheckInterval = Number(process.env.SOCKETCLUSTER_BOOT_CHECK_INTERVAL) ||
 var bootTimeout = Number(process.env.SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT) || 10000;
 var bootStartTime = Date.now();
 
-var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} ` +
-`before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
+var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
 
 waitForFile(masterControllerPath, bootCheckInterval, bootStartTime, bootTimeout, errorMessage)
-.catch((err) => {
-  console.error('> Boot error: ' + err.message);
-  process.exit(1);
-});
+  .catch((err) => {
```

Comment:
Should I exclude any changes made by autobeautifier from comments or is it fine to include them?
Or, maybe, it's better extract such changes into separate commits?

Reply:
Yes, better exclude them because it's against the current convention. Right now there is no indentation if the line starts with a dot.